package.json

```diff
@@ -59,29 +59,33 @@
     "types/*.d.ts"
   ],
   "dependencies": {
-    "@babel/code-frame": "^7.10.4",
-    "@babel/runtime": "^7.12.5",
-    "@types/aria-query": "^5.0.1",
+    "@babel/code-frame": "7.18.6",
+    "@babel/runtime": "7.18.9",
+    "@types/aria-query": "5.0.1",
     "aria-query": "5.3.0",
-    "chalk": "^4.1.0",
-    "dom-accessibility-api": "^0.5.9",
-    "lz-string": "^1.5.0",
-    "pretty-format": "^27.0.2"
+    "dom-accessibility-api": "0.5.9",
+    "lz-string": "1.5.0",
+    "picocolors": "1.1.1",
+    "pretty-format": "27.0.2",
+    "typescript": "4.5"
```

Comment:
`typescript` needs to be listed as a `devDependency`, or is there a reason to move it into the dependencies?

Reply:
Just an oversight and probably a leftover from trying to get the versions right :D